src/components/zoom-buttons/zoom-buttons.tsx

```diff
@@ -14,19 +14,21 @@ export const ZoomButtons = (props: IZoomButtonsProps) => {
     >
       <div className="zoom-button-container">
         <div
+          data-testid="zoom-in-button"
           className="zoom-button"
-          onClick={handleZoomOut}
-        >
-          <MinusIcon />
+          onClick={handleZoomIn}
+          >
```

Comment:
I think you mean for this closing bracket to be back one tab, but that's beyond nitpicking at this point